helm/etcd-cluster-migrator/templates/job-psp.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ .Values.name }}-job
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.name }}-job
+    giantswarm.io/service-type: "managed"
```

Comment:
We use `giantswarm.io/managed-by` when it is managed by us. See also https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels.

Reply:
ack, adding `giantswarm.io/managed-by: "aws-operator"` for now